arangod/RestServer/DatabaseFeature.cpp

```diff
@@ -956,21 +950,19 @@ void DatabaseFeature::enumerateDatabases(
 }
 
 void DatabaseFeature::updateContexts() {
-  TRI_ASSERT(_vocbase != nullptr);
+  auto* vocbase = useDatabase(TRI_VOC_SYSTEM_DATABASE);
+  TRI_ASSERT(vocbase);
```

Comment:
Can we move this getting of the `vocbase` further down?
Because if V8 is disabled, this method wil return a few lines later (line 959) and the vocbase is used, but will not be release. This is effectivey a leak, though it is only for the system database.

Reply:
sure